k8s/helm-chart/dynatrace-gcp-function/values.yaml

```diff
@@ -84,6 +94,8 @@ importDashboards: "true"
 # Import predefined alerting rules for selected services. Allowed values: "true"/"yes", "false"/"no"
 importAlerts: "true"
 # configuration file for GCP services - uncommented services will be monitored
+#clusterName Cluster Name, should be empty as it's filed by setup.sh.
```

Comment:
what does it mean? I didn't find setting this up in setup.sh

Reply:
Fixed file reference.